.gitignore

```diff
@@ -39,3 +40,4 @@ callgrind.out.*
 tests/autotests/autotests
 .ycm_extra_conf.py*
 *.stash
+
```

Comment:
> Do you have some link for your standard?

As far as standards go.. the single newline is used for concatenation on file streams. So there are a few out there. :) ... but as I already commented the editor on the Mac I used bites big time especially since it hid it from me. Saw it in Linux here so it's removed.